cockpit/src/main/java/com/polidea/cockpit/event/PropertyChangeListener.kt

```diff
@@ -0,0 +1,5 @@
+package com.polidea.cockpit.event
+
+interface PropertyChangeListener<T> {
```

Comment:
We have to write this interface in Java. The first reason is we want to have pure Java `Cockpit` class, so `PropertyChangeListener` should be written in Java also. The second reason are Kotlin's SAM conversions https://kotlinlang.org/docs/reference/java-interop.html#sam-conversions that work for Java interfaces only (unfortunately)